server/swissvoice/static/css/landingpage.css

```diff
@@ -1,92 +1,126 @@
 /*here is all the CSS for the actual landingpage*/
 
-#action-section{
-  max-width: 700px;
+#action-section {
+    max-width: 700px;
 
 }
 
-.action-text{
-  line-height: normal;
+.action-text {
+    line-height: normal;
+}
+
+.card-body.fixed-size {
+    align-self: start;
 }
 
 .jumbotron {
-  background-color: #d6f1ff; /*color action-section*/
+    background-color: #d6f1ff; /*color action-section*/
 }
 
 #main-schreiben-btn {
-  font-size: 1.5rem;
-  background-color:  #63aeff; /*color of button in action-section*/
-  border-color: transparent;
+    font-size: 1.5rem;
+    background-color: #63aeff; /*color of button in action-section*/
+    border-color: transparent;
 }
 
 #main-schreiben-btn:hover {
-  background-color:  #a2a0ff;
-  transition: all 0.5 ease;
+    background-color: #a2a0ff;
+    transition: all 0.5s ease;
 }
 
 #sv-info-text-card {
 }
 
 .sv-info-header {
-  background-color: transparent;
+    background-color: transparent;
+}
+
+.sv-info-text-title {
+    border-color: transparent;
+    font-weight: 400;
 }
 
-#sv-info-text-title{
-  border-color: transparent;
-  font-weight: 400;
+.sv-info-text:first-of-type {
+    margin-top: 1rem;
 }
 
-#sv-info-tx1 {
-  text-align: left;
-  line-height: normal;
-  height: 300px; /*fallback*/
-  overflow-y: scroll;
-  /*box-shadow: 0 4rem 3px 3px rgba(213, 236, 237, 0.5) inset;*/
+.sv-info-text {
+    text-align: left;
+    line-height: normal;
+    margin-bottom: 0;
+    transition: height 1s;
+    overflow: hidden;
 }
 
-#sv-info-tx2 {
-  text-align: left;
-  line-height: normal;
+.teaser {
+    position: relative;
+    overflow: hidden;
+}
+
+.teaser::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0), #d6f1ff);
+    background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0)), to(#d6f1ff));
+    background-image: -webkit-linear-gradient(top, rgba(255, 255, 255, 0), #d6f1ff);
+    background-image: -moz-linear-gradient(top, rgba(255, 255, 255, 0), #d6f1ff);
+    background-image: -o-linear-gradient(top, rgba(255, 255, 255, 0), #d6f1ff);
+    background-image: -ms-linear-gradient(top, rgba(255, 255, 255, 0), #d6f1ff);
+}
+
+.teaser + .teaser-trigger::after {
+    content: "Weiterlesen";
+}
+
+:not(.teaser) + .teaser-trigger {
+    margin-top: 1rem;
+}
+
+:not(.teaser) + .teaser-trigger::after {
+    content: "Nicht mehr Weiterlesen?? Plz improve";
```

Comment:
We **might** want to improve this little text over here tho

Reply:
maybe: "verkürzen" or "vermindern" or "verringern"
-->personally "verringern" is my fav.